extutils/imgproc/apng2png.py

```diff
@@ -0,0 +1,357 @@
+"""
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the first line (D212)](https://app.codacy.com/manual/RaenonX/Jelly-Bot/pullRequest?prid=6015490)